scripts/start-containers.sh

```diff
@@ -102,19 +121,28 @@ function runintegrationtests {
 
 }
 
-docker-machine start docker-dev
-eval $(docker-machine env docker-dev)
+if hash docker-machaine 2> /dev/null
+then
+    echo "Starting docker machine"
+    docker-machine start docker-dev
+    eval $(docker-machine env docker-dev)
+
+    export HostIP=$(docker-machine ip docker-dev)
+    # Set HostIP to localhost if docker-machine doesn't run
+    if [ $HostIP="" ]; then
+        export HostIP="127.0.0.1"
+    fi
+fi
 
-docker pull mierdin/todd:$branch
+image=mierdin/todd:$branch
```

Comment:
I like the idea:

- Can you move this to the top?
- Can you name the var something todd specific? I.e. ```toddimage```.
- I am also statically referring to ```mierdin/todd:$branch``` several other places in this script. Would be cool if those could be replaced with this var as well. That was my intention with the ```branch``` var on line 12 but I just didn't extend it to the full image name.